packages/mui-styled-engine-sc/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/styled-engine-sc",
-  "version": "5.1.0",
+  "version": "5.1.1",
```

Comment:
FYI, I manually update this.

Reply:
What was the reason? Was there a change not picked up by lerna?

Reply:
At first I found it strange that `style-engine` is bumped but `styled-engine-sc` does not. But I found that emotion is bumped, so I will revert this change on `styled-engine-sc`.